### PR TITLE
(SEC) IronHummer unique faction items update

### PR DESCRIPTION
**Pull request by DarkLevelSS13**
## About The Pull Request
Adding two unique faction items for (SEC)IronHummer.
1)Sky Driver. Old, Syndicate revolver made on lost tech before the Corporate war. Loaded with .35 caliber, have a bit increased damage, but due use of special gravitational generator assisted barrel, allows armor penetration of 100 on this caliber. You don’t want to lose this thing to Vagabond. Can shot through walls and people with easy. Can hold 5 ammo.
image
2) Bluespace Biosyphon. Hunts on flora and fauna that sometimes populates bluespace, and use them to produce donuts endlessly. Powered by nuclear generator, this thing will output 1 box of donuts each 15 minutes.
image
Also, adding id lock for blast doors.

## Why It's Good For The Game
Those are needed for new individual objects, and will serve as new traitor targets as well, because contracts will now prioritize stealing more.

Those items cannot be reproduced in RnD, but they do provide a lot of research points.

## Changelog
:cl: DarkLevel
add: Adding new item Bluespace Biosyphon.
add: Adding new gun Sky Driver.
add: Id lock for blast doors.
code: Adding possibility to coding wall penetration mods for guns.
/:cl:

:cl: infrared_baron
imageadd: Make sprites for Bluespace Biosyphon.
imageadd: Make sprites for Sky Driver.
/:cl:

### CODE REVIEW

**Comment by ArmolitskiyCarpoeb** [comment]
show sprites

File: code/modules/projectiles/guns/projectile/revolver/sky_driver.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	item_state = "sky_driver"
+	drawChargeMeter = FALSE
+	max_shells = 5
+	caliber = CAL_PISTOL
+	origin_tech = list(TECH_COMBAT = 10, TECH_MATERIAL = 2)
+	ammo_type = /obj/item/ammo_casing/pistol
+	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_WOOD = 6)
+	price_tag = 20000
+	damage_multiplier = 1.5 //because pistol round
+	penetration_multiplier = 20
```
Comment (MadmanMartian) [inline code/modules/projectiles/guns/projectile/revolver/sky_driver.dm:15]: what the frick

File: code/modules/projectiles/guns/projectile/revolver/sky_driver.dm [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	icon_state = "sky_driver"
+	item_state = "sky_driver"
+	drawChargeMeter = FALSE
+	max_shells = 5
+	caliber = CAL_PISTOL
+	origin_tech = list(TECH_COMBAT = 10, TECH_MATERIAL = 2)
+	ammo_type = /obj/item/ammo_casing/pistol
+	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_WOOD = 6)
+	price_tag = 20000
+	damage_multiplier = 1.5 //because pistol round
```
Comment (MadmanMartian) [inline code/modules/projectiles/guns/projectile/revolver/sky_driver.dm:14]: what the frick

Comment (DarkLevelSS13) [reply code/modules/projectiles/guns/projectile/revolver/sky_driver.dm:14]: What do you want to know?

Comment (SpadesSlick) [reply code/modules/projectiles/guns/projectile/revolver/sky_driver.dm]: Yeah, that's too much. Reduced.

File: code/modules/projectiles/guns/projectile/revolver/sky_driver.dm (hunk 1 above)
Comment (SpadesSlick) [reply code/modules/projectiles/guns/projectile/revolver/sky_driver.dm:15]: That's gun gimmick, a piercing capability to work as AMR with weak ammo.

**Comment by Kurgis** [comment]
@drexample Does the Changelog bot support multiple changelogs within a single PR like what was done above?

File: code/datums/objective/steal.dm [hunk 3]
```diff
@@ -22,7 +22,9 @@
 		"the hypospray" = /obj/item/weapon/reagent_containers/hypospray,
 		"the captain's pinpointer" = /obj/item/weapon/pinpointer,
 		"an ablative armor vest" = /obj/item/clothing/suit/armor/laserproof,
-		"an Ironhammer hardsuit control module" = /obj/item/weapon/rig/combat/ironhammer
+		"an Ironhammer hardsuit control module" = /obj/item/weapon/rig/combat/ironhammer,
+		"a Bluespace Biosyphon" = /obj/item/biosyphon,
+		"a FS REV .35 Auto \"Sky Driver\"" = /obj/item/weapon/gun/projectile/revolver/sky_driver
```
Comment (SpadesSlick) [inline code/datums/objective/steal.dm:27]: Name is outdated, it's not FS revolver.

File: code/game/objects/items/devices/biosyphon.dm [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	icon = 'icons/obj/faction_item.dmi'
+	icon_state = "biosyphon"
+	item_state = "biosyphon"
+	w_class = ITEM_SIZE_BULKY
+	flags = CONDUCT
+	throwforce = WEAPON_FORCE_PAINFUL
+	throw_speed = 1
+	throw_range = 2
+	price_tag = 20000
+	origin_tech = list(TECH_MATERIAL = 4, TECH_BLUESPACE = 9, TECH_POWER = 7)
```
Comment (SpadesSlick) [inline code/game/objects/items/devices/biosyphon.dm:13]: Tech_bluespace 9 is clearly overkill, 8 will be fine.

File: code/modules/projectiles/guns/projectile/revolver/sky_driver.dm [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+	caliber = CAL_PISTOL
+	origin_tech = list(TECH_COMBAT = 10, TECH_MATERIAL = 2)
+	ammo_type = /obj/item/ammo_casing/pistol
+	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_WOOD = 6)
+	price_tag = 20000
+	damage_multiplier = 1.1 //because pistol round
+	penetration_multiplier = 20
+	pierce_multiplier =  5
+	recoil_buildup = 50
+	gun_tags = list(GUN_PROJECTILE, GUN_CALIBRE_35)
```
Comment (SpadesSlick) [inline code/modules/projectiles/guns/projectile/revolver/sky_driver.dm:18]: Please add a newline and make sure that your tags are work appropriately after #4994.

**Pull request merged**